spec/models/invoice_spec.rb

```diff
@@ -21,18 +21,18 @@
         item_5 = create(:item, merchant: merchant)
         item_6 = create(:item, merchant: merchant)
 
-        invoice_1 = create(:invoice, customer: customer) # 2 items that have not shipped
-        invoice_2 = create(:invoice, customer: customer) # 1 item shipped, 1 not shipped
-        invoice_3 = create(:invoice, customer: customer) # 2 items that have shipped
+        invoice_1 = create(:invoice, created_at: 3.day.ago, customer: customer) # 2 items that have not shipped
+        invoice_2 = create(:invoice, created_at: 2.day.ago, customer: customer) # 1 item shipped, 1 not shipped
+        invoice_3 = create(:invoice, created_at: 2.day.ago, customer: customer) # 2 items that have shipped
 
         invoice_item_1 = create(:invoice_item, status: "pending", item: item_1, invoice: invoice_1)
         invoice_item_2 = create(:invoice_item, status: "packaged", item: item_2, invoice: invoice_1)
         invoice_item_3 = create(:invoice_item, status: "pending", item: item_3, invoice: invoice_2)
         invoice_item_4 = create(:invoice_item, status: "shipped", item: item_4, invoice: invoice_2)
         invoice_item_5 = create(:invoice_item, status: "shipped", item: item_5, invoice: invoice_3)
         invoice_item_6 = create(:invoice_item, status: "shipped", item: item_6, invoice: invoice_3)
-        
-        expect(Invoice.find_incomplete_invoices).to eq([invoice_2, invoice_1])
+
+        expect(Invoice.find_and_sort_incomplete_invoices).to eq([invoice_1, invoice_2])
       end
     end
 
```

Comment:
Delete wild blank line.

Reply:
I realize this wasn't done on this PR but mentioned it since you're already here.

Reply:
Done and done!